pom.xml

```diff
@@ -25,11 +25,12 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- JAVA version -->
-        <java.version>1.8</java.version>
+        <java.version>11</java.version>
 
         <!-- dependencies -->
 
-        <aspectj.version>1.8.8</aspectj.version>
+        <aspectj.version>1.9.4</aspectj.version>
```

Comment:
Latest AspectJ is now 1.9.6 (according to dependabot)

Reply:
Bumped, thanks

Reply:
I merged #676 the latest is 1.9.6.